### PR TITLE
P1/feat/64 add toplist in carousel

### PR DESCRIPTION
**Pull request by netlight-johannadagfalk**

### CODE REVIEW

File: app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselCategoryButton.jsx [hunk 1]
```diff
@@ -31,26 +36,29 @@ const CarouselCategoryButton = ({
...
+        (temp) => temp.climate_action_category_id === categoryID
+      );
+    }
+    updateCategoryAttributes(filteredActions);
   };
 
-  const allCategoriesClick = () => {
-    setAllCategories(true);
-    updateCategory();
+  const categoryClick = (popularValue, allCategoriesValue, categoryID) => {
```
Comment (elinaolsson) [inline app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselCategoryButton.jsx:55]: Is this a function we should name "handleCategoryClick"?

File: app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselCategoryButton.jsx [hunk 2]
```diff
@@ -31,26 +36,29 @@ const CarouselCategoryButton = ({
...
+    setPopular(popularValue);
```
Comment (elinaolsson) [inline app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselCategoryButton.jsx:56]: If this is a boolean we should name it something like "showPopular" or "popularClicked"

Comment (elinaolsson) [reply app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselCategoryButton.jsx:56]: Same with allCategoriesValue

**Review by elinaolsson — APPROVED** [review]
🥳

**Review by netlightebba — APPROVED** [review]

**Pull request merged**